app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -106,4 +106,44 @@ public function store(Request $request, $seatId = null, $backTo = null)
         // Redirect to the license page with error
         return redirect()->route('licenses.index')->with('error', trans('admin/licenses/message.checkin.error'));
     }
+
+    public function checkinCount($license_id){
+        $licenseSeats = LicenseSeat::where('license_id', '=', $license_id)
+            ->whereNotNull('assigned_to')
+            ->with('user')
+            ->get();
+
+        return('There are '.$licenseSeats->count().' seats checked out. Are you sure you want to check all of them in?');
```

Comment:
This looks a little funny to me. If we're only going to return a message, the response should probably be in our standard JSON response, and should be coming from the API controller. We don't typically just return text in a normal controller.